docs/usage/errors.md

```diff
@@ -1,6 +1,7 @@
 Below are details on common errors developers can see when working with pydantic, together
 with some suggestions on how to fix them.
 
+{% raw %}
```

Comment:
what do these "raw" tags do?

Reply:
Jinja2 is needed to handle the markdown embed, but as a side effect it interprets the `{#` used to specify anchor links as comments and fails. The `{% raw %}` tag fences this page and jinja effectively ignores it.